cmd/run.go

```diff
@@ -40,17 +41,19 @@ func (r *runJob) run(cmd *cobra.Command, args []string) {
 	if !verbose {
 		log.SetLevel(log.WarnLevel)
 	}
+	if r.cleanup != job.All.String() && r.cleanup != job.Succeeded.String() && r.cleanup != job.Failed.String() {
+		err := errors.New("please set 'all', 'succeeded' or 'failed' as --cleanup.")
```

Comment:
__[Golint]__ error strings should not be capitalized or end with punctuation or a newline ([link](https://sider.review/gh/repos/169710725/pulls/25#issue-4259317))

<sub>posted by Sider</sub>